### PR TITLE
Prepared input files for fragments/jak2_set1 system

### PR DESCRIPTION
**Pull request by BillSwope**
## Input submission checklist

* [x ] created a directory for each system based on the [template directory][templates]
* [x ] named and placed the directory with the following pattern: `input_structures/prepared_structures/<set_name>/<system_name>`

For each submitted directory:

* [x ] filled in system preparation details in the `PREPARATION_DETAILS.md` file
* [x] added a PDB file with the protein named `protein.pdb`
* [x ] removed any cofactors from the PDB file and placed them in `cofactors.sdf`
* [x ] did not re-protonate the residues, but left protonation state of the protein unchanged
* [x] kept cystallographic waters and metals in the PDB file
* [x ] tested the PDB and, if available, cofactors.sdf file using the [input validation script][input_validation]
* [x ] copied over the ligands SDF file to a file named `ligands.sdf` and removed any duplicate binding modes or protonation states, keeping the more favorable state

## Summary of changes
Protein: added NME/ACE capping groups; changes phosphate from tyrosine (PTR -> TYR); removed one water that was clashing with NME; rename NMA->NME.
Ligand: removed two ligands: 4b and "3 althernative".  4b was an alternate protonation state of 4a, which had the stronger binding energy; "3 alternative" was a different binding pose of "3", but Schrodinger did not report a dG for this "3 alternative"

## Licensing agreement
* [x ] I agree to release these inputs under the combined MIT and CCBY SA 4.0 licenses of this repository

[templates]: https://github.com/OpenFreeEnergy/IndustryBenchmarks2024/tree/main/industry_benchmarks
[input_validation]: https://github.com/OpenFreeEnergy/IndustryBenchmarks2024/tree/main/industry_benchmarks/utils/input_validation.py

### CODE REVIEW

File: industry_benchmarks/input_structures/prepared_structures/fragments/jak2_set1/protein.pdb
Comment (hannahbaumann) [inline industry_benchmarks/input_structures/prepared_structures/fragments/jak2_set1/protein.pdb:1]: I noticed that for the two CYS residues 917 and 961 there are two alternate conformations. Could you potentially remove one of them such that the pdb has only a single conformation?

**Review by hannahbaumann — CHANGES_REQUESTED** [review]
Thanks @BillSwope , this looks good to me, just the one comment!

**Comment by BillSwope** [comment]
Sure, thanks!  I didn't even notice this!  I don't suppose it matters which
conformation I take, right?

What about the jak2_set1 that Binqing submitted?  I have not seen it.

Regards...Bill

**Comment by hannahbaumann** [comment]
Thanks @BillSwope , I noticed that there are additional residues with alternate conformations: Val916, ILE982, Arg1117. The pdb file from Schrodinger contains the occupancy of the two states (the column after the z coordinate), could you keep the conformation with the higher occupancy (e.g. for VAL 916 the A-state is 65% while B-state is 35%).
Binqing only kept a single conformation, however there are some other changes needed for that preparation (e.g. keeping crystallographic waters). Maybe you could talk about it and let us know which submission we should include here?

**Review by hannahbaumann — APPROVED** [review]
Thanks @BillSwope , this looks good to me!

**Review by IAlibay — APPROVED** [review]
Thanks @BillSwope !

**Pull request merged**